addon/maptools/measuretool.js

```diff
@@ -0,0 +1,39 @@
+import MapTool from 'ember-flexberry-gis/maptools/maptool';
+
+export default MapTool.extend({
+  _layer: null,
+  _measure: null,
+
+  isMultiTool() {
+    return true;
+  },
+
+  measureClose(e) {
+    this._layer.disableEdit();
+    this.get('map').off('measure:created');
+    this._layer.remove();
+  },
+
+  measured(e) {
+    this._layer = e.layer;
+    this.get('map').off('measure:created');
+  },
+
+  enable() {
+    this._super(...arguments);
+    // there are problems if dragging is disabled
+    var map = this.get('map');
+    map.dragging.enable();
+    this._measure = map.MeasureTools ? map.MeasureTools : L.measureBase(map);
```

Comment:
map.MeasureTools is initialized in L.measureBase(map) (similar as properties map.editTools in Leaflet.Measure)